### ISSUE TITLE
OCA cause Legacy Update 1.8 unable to Update Windows XP

### ISSUE THREAD

**Issue by sal-ammoniac** [kirb/LegacyUpdate#196]
https://github.com/Skulltrail192/One-Core-API-Binaries/issues/205

1.7.1 did not have this issue

**Comment by kirb** [comment]
Interesting... 80248011 is "Could not create a data store object in another process." Your log also shows that the underlying error is

```
FATAL: Error: 0xc8000426. wuauclt datastore: failed to spawn COM server
FATAL: DS: Out of proc datastore process exited with error 0xc8000426 before signalling ready event.
```

Seems like `wuauclt.exe` is constantly trying to launch and is crashing, based on the "logging initialised" message being repeated a bunch of times. Not sure why that would work with 1.7.1 but not 1.8. Doesn't feel like it should be related to LU at all... but hard to tell from the outset without some further scenario testing which I'll try at some point (if someone doesn't beat me to it).

**Comment by sal-ammoniac** [comment]
problem persists on 1.9

**Comment by kirb** [comment]
I reproduced this issue, but it's not LU specific at all. `wuauclt.exe /RunStoreAsComServer` repeatedly crashes in the background with OCA Base installed, meaning Automatic Updates doesn't work either. I can't find any relation to LU causing/contributing to the issue - this happens with 1.7.1 installed, and also in the background with it uninstalled.

Not sure what to do about it from here, it seems like something @Skulltrail192 needs to look into, so closing this issue.

**Comment by sal-ammoniac** [comment]
fixed in OCA 3.0.4
